### PR TITLE
Add the Cloudstack updateVmNicIp call

### PR DESCRIPTION
**Pull request by mcorbin**
This call allows to modify the static ip associated to a nic.

```
$ ./exo vm updatenicip 2269764c-39ef-43a4-9a97-dc17a155bb2e 10.0.0.5
Updating the nic "2269764c-39ef-43a4-9a97-dc17a155bb2e" . success.
Instance ID:           8f0d4701-dce1-432c-87bd-5548c0860a43
Isntance Hostname:     test-vm
Instance Display name: test-vm
┼──────────────────────────────────────┼──────────────────────────────────────┼────────────┼
│                NIC ID                │              NETWORK ID              │ IP ADDRESS │
┼──────────────────────────────────────┼──────────────────────────────────────┼────────────┼
│ 2269764c-39ef-43a4-9a97-dc17a155bb2e │ 932e1e8e-9a07-4449-8894-b0ad52fcec6d │ 10.0.0.5   │
┼──────────────────────────────────────┼──────────────────────────────────────┼────────────┼
```

### CODE REVIEW

File: cmd/vm_update_nic_ip.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	resp, err := asyncRequest(req, fmt.Sprintf("Updating the nic %q ", nicID))
+	if err != nil {
+		return err
+	}
+
+	newVM := resp.(*egoscale.VirtualMachine)
+
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', tabwriter.TabIndent)
+	fmt.Fprintf(w, "Instance ID:\t%s\n", newVM.ID)                    // nolint: errcheck
+	fmt.Fprintf(w, "Isntance Hostname:\t%s\n", newVM.Name)            // nolint: errcheck
```
Comment (chrisglass) [inline cmd/vm_update_nic_ip.go]: s/Isntance/Instance/ here

**Comment by mcorbin** [comment]
New parameters : `<vm_name | id> <network_name | id> <nic_ip>`

```
$ ./exo vm updatenicip test-vm managed-privnet 10.0.0.11
Updating the nic "2269764c-39ef-43a4-9a97-dc17a155bb2e". success.
Instance ID:           8f0d4701-dce1-432c-87bd-5548c0860a43
Instance Hostname:     test-vm
Instance Display name: test-vm
┼──────────────────────────────────────┼──────────────────────────────────────┼────────────┼
│                NIC ID                │              NETWORK ID              │ IP ADDRESS │
┼──────────────────────────────────────┼──────────────────────────────────────┼────────────┼
│ 2269764c-39ef-43a4-9a97-dc17a155bb2e │ 932e1e8e-9a07-4449-8894-b0ad52fcec6d │ 10.0.0.11  │
┼──────────────────────────────────────┼──────────────────────────────────────┼────────────┼
```

File: cmd/vm_update_nic_ip.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		if err != nil {
+			return err
+		}
+		network, err := getNetworkByName(args[1])
+		if err != nil {
+			return err
+		}
+		nic := vm.NicByNetworkID(*network.ID)
+		if nic == nil {
+			return fmt.Errorf("The virtual machine %s is not associated to the network %s", vm.DisplayName, network.Name)
```
Comment (pierre-emmanuelJ) [inline cmd/vm_update_nic_ip.go]: First capital letter is not conventional for an error message

**Comment by mcorbin** [comment]
Thanks for the reviews. I updated the PR.

**Comment by greut** [comment]
How do you remove the static lease?

**Comment by greut** [comment]
going full yaml :robot: 

```console
% exo vm updateip yoyo test 10.0.0.42

updating the static lease of NIC "279da692-5b6f-4eff-88af-2e3b77853ac2". success.

Instance ID: e540307e-9b3e-4300-9848-bd049d2489ba
Name:        yoyo
Network Interfaces:
- Network:    Public
  IP Address: 159.100.241.244
- Network:    test
  IP Address: 10.0.0.37

```

**Comment by greut** [comment]
@pierre-emmanuelJ if you're happy with it, feel free to merge.

**Pull request merged**